rules/do-you-use-gravatar-for-your-profile-pictures/rule.md

```diff
@@ -1,29 +1,29 @@
 ---
 type: rule
-archivedreason: 
-title: Do you use Gravatar for your profile pictures?
-guid: b790dbfc-fe37-4a4a-992e-c9294620a06b
+title: Do you use Gravatar(or Cravatar for China) for your profile pictures?
 uri: do-you-use-gravatar-for-your-profile-pictures
-created: 2015-03-16T10:17:40.0000000Z
-authors: 
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects: []
-
+created: 2015-03-16T10:17:40.000Z
+archivedreason: null
+guid: b790dbfc-fe37-4a4a-992e-c9294620a06b
 ---
-
 To keep profile management simple and make it easier for users to have a consistent experience across applications, you should use [Gravatar](https://en.gravatar.com/) for showing profile images in your application.
 
-Your Gravatar is an image that follows you from site to site appearing beside your name when you do things like comment or post on a blog.
+But Gravatar is blocked by Great Firewall of China (GFW), and clients from China can't access their profile photos successfully without a VPN. There is an alternative option for Chinese users which is [Cravatar](https://cravatar.cn/).
```

Comment:
Hi @babakamyljanovssw 

- [ ] 1. Please add this to a China box https://www.ssw.com.au/rules/rule/#5-boxes
- [ ] 2. Remove the word "but" because it's negative